string state
bool target_acquired
float32 target_centroid_x
float32 target_centroid_y
int32 target_width
int32 target_height
int32 target_xo
int32 target_yo
int64 target_area

